superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -596,10 +598,32 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     }
   }, [availableDbs]);
 
+  useEffect(() => {
+    if (validationErrors) {
+      setHasErrors(true);
+    } else {
+      setHasErrors(false);
+    }
+  }, [validationErrors]);
+  // Used as componentDidMount
+
   const tabChange = (key: string) => {
     setTabKey(key);
   };
 
+  const errorAlert = () => {
+    const errors = validationErrors;
+    return (
+      <Alert
+        type="error"
+        css={(theme: SupersetTheme) => antDErrorAlertStyles(theme)}
+        message="Missing Required Fields"
```

Comment:
Not sure if you plan to put this into the v2, but I think that we're going to have to create a map between the server side error and the language that we want to show in these alerts. These are more descriptive and instructional and I don't think they will be or should be the same as what we are returning from the API.

Reply:
I hadn't thought about a map, though I really like that. 

No, I figured that language was prone to change, I wanted this in there to make sure that it looked ok, and so that I could style it.